.github/workflows/msi.yaml

```diff
@@ -72,9 +72,8 @@ jobs:
         with:
           path: |  # TODO: go build cache if we care, noting it is OS-specific
```

Comment:
I will sort out the build cache in a different PR as it requires handling Os-specifics!